server/models/recipe.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+
+	id: mongoose.Schema.Types.ObjectId,
+	name: { type: String, required: true },
+	ingredients: { type: String, required: true },
+	description: { type: String, required: true },
+	price: { type: Number, required: true },
+	chef: { type: Schema.Types.ObjectId, ref: 'User', required: true },
+	servingSize: { type: String, required: true },
+	requiredStuff: { type: String, required: true },
+	picture: { type: String },
```

Comment:
what we're missing here that is in the mockups is `cuisine`, for which we can hardcode some options, you can just pick them up from the mocks